src/MultiText/MultiText.js

```diff
@@ -11,7 +11,7 @@ import React from "react";
 export default function LabelSetter({
   onChange = () => {},
   rows = [],
-  style = { height: 250, width: "100%" }
+  style = { height: 254, width: "100%" }
```

Comment:
Why is this seemingly random 254 pixels the default style? In fact, why do I need to pass style at all?